ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerSubscription.scala

```diff
@@ -169,13 +170,11 @@ object ParallelIndexerSubscription {
       lastRecordTime = 0,
       batch = Vector.empty,
       batchSize = 0,
-      averageStartTime = 0,
       offsets = Vector.empty,
     )
 
   def seqMapper(
-      internize: Iterable[DbDto] => Iterable[(Int, String)],
-      metrics: Metrics,
+      internize: Iterable[DbDto] => Iterable[(Int, String)]
```

Comment:
On the other hand here with the seqMapper I would love to have some metrics. This is a sequential phase which is running on akka threads, would love to see if this spins out of control!
WDYT @tudor-da @mziolekda

Reply:
The point here when we discussed with @tudor-da that this stage is almost noop

Reply:
Yeah, my omission here. Since interning appeared quite recently, probably it's something that we should keep an eye on.

Reply:
I second @tudor-da on this one, metrics will be generated on a batched resolution anyway, let measure this stage (needs manual metrics duration addition here, like you already did)